Test/datatypes/t4.bpl

```diff
@@ -0,0 +1,42 @@
+// RUN: %parallel-boogie "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+type{:datatype} Pair;
+function{:constructor} Pair(a: int, b: int): Pair;
```

Comment:
What's the benefit of `function{:constructor} Pair(a: int, b: int): Pair;` over `constructor Pair(a: int, b: int): Pair;` ?

Reply:
See my response to your first comment.

Multiple constructors are allowed and used. There are many such tests in the civl folder.

Reply:
Sounds good, thanks :)